site/tutorials/tutorial-two-java.md

```diff
@@ -231,8 +231,8 @@ the consumer dies. It's fine even if processing a message takes a very, very
 long time.
 
 Message acknowledgments are turned on by default. In previous
-examples we explicitly turned them off via the `autoAck=true`
-flag. It's time to remove this flag and send a proper acknowledgment
+examples we explicitly turned them off passing `true` to `autoAck` flag of method
+`basicConsume`. It's time to remove this flag and send a proper acknowledgment
```

Comment:
I tend to prefer the explicit method, so the wording would be _It's time to set this flag to `false` and ..._